apps/portals/src/utils.ts

```diff
@@ -12,3 +12,24 @@ export const DESKTOP_VIEWPORT_MIN_WIDTH_MEDIA_QUERY = `(min-width: ${DESKTOP_VIE
 
 export const useShowDesktop = () =>
   useMedia(DESKTOP_VIEWPORT_MIN_WIDTH_MEDIA_QUERY)
+
```

Comment:
May move these functions back into SRC or a different app utils project when we have multiple apps in the monorepo and we want to refactor duplicate code